src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/PropertyPages/DebugPageViewModel.cs

```diff
@@ -394,6 +414,7 @@ private bool TrySetOtherProperty<T>(string propertyName, T value, T defaultValue
 
         public bool SupportNativeDebugging       => ActiveProviderSupportsProperty(UIProfilePropertyName.NativeDebugging);
         public bool SupportSqlDebugging          => ActiveProviderSupportsProperty(UIProfilePropertyName.SqlDebugging);
+        public bool SupportJSWebView2Debugging => ActiveProviderSupportsProperty(UIProfilePropertyName.JSWebView2Debugging);
```

Comment:
Please fix the indentation here.

```suggestion
        public bool SupportJSWebView2Debugging   => ActiveProviderSupportsProperty(UIProfilePropertyName.JSWebView2Debugging);
```